schedule/yast/opensuse/installer_extended/installer_extended.yaml

```diff
@@ -15,12 +15,10 @@ schedule:
   - installation/installation_mode
   - installation/logpackages
   - installation/system_role/select_role_desktop_with_KDE_plasma
-  - installation/partitioning/select_guided_setup
-  - installation/partitioning/guided_setup
```

Comment:
I don't know why here the test is using Guided Setup. I checked logs and video and noticed that it does not change anything there, so I guess we can skip entering to Guided Setup, as everything remains by default anyway.